contributors/guide/first-contribution.md

```diff
@@ -99,4 +138,22 @@ Check the [issue triage guide](./issue-triage.md) for more information.
 
 *Example:* a documentation issue should be opened to [kubernetes/website](https://github.com/kubernetes/website/issues).
```

Comment:
This section looks like it might have been missed :x 
It still needs formatting and link fixes.

Reply:
Done!